antennas-postgres/README.md

```diff
@@ -107,7 +108,7 @@ GRANT SELECT ON antennas, antennas_performance TO materialize;
 
 
   -- Turn the Postgres tables into Materialized Views
-  CREATE MATERIALIZED VIEWS FROM SOURCE antennas_publication_source;
+  CREATE VIEWS FROM SOURCE antennas_publication_source;
```

Comment:
Having another look, we could also drop the `MATERIALIZED` bit in the source and completely remove that mental overhead. In the end, we're only creating a single materialization.

Reply:
Omw.